src/Components/App.tsx

```diff
@@ -557,6 +566,55 @@ export function App({
   }
 }
 
+// This function helps launch apps exported with the shinylive Python and R
+// packages and is used by `export_template/index.html`.
+export async function runExportedApp(
+  id: string,
+  appEngine: AppEngine,
+  relPath = "",
```

Comment:
Please update to use deconstructed params, and not ordered arguments.  

This helps with parameters being added/removed down the road.